security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -993,14 +993,16 @@ func TestDebugEndpoints(t *testing.T) {
 		arg := Options{
 			EnableIngressGatewaySDS: false,
 			EnableWorkloadSDS:       true,
-			RecycleInterval:         2 * time.Second,
+			RecycleInterval:         30 * time.Second,
```

Comment:
So the is secret is recycled before the test finishes? Can you just set it to 0?